src/configs/index.js

```diff
@@ -1,46 +1,8 @@
+export const language = 1;
 export const itemsPerPage = [15, 30, 60];
 export const PRODUCT_LIST_URL = '/productlist/';
 export const CATALOGS_URL = '/catalogs/';
 export const placeholder = 'Type here...';
-export const countries = {
-  title: 'Country',
-  name: 'country',
-  value: [
-    '',
-    'Ukraine',
-    'Italy',
-    'Netherlands',
-    'Poland',
-    'Portugal',
-    'France',
-    'Germany',
-    'Greece',
-    'Spain',
-    'Hungary',
-    'Sweden',
-    'England',
-    'USA'
-  ]
-};
-export const paymentMethods = {
-  title: 'Payment Method',
-  name: 'paymentMethod',
-  value: [
-    '',
-    'credit card',
-    'pay pal',
-    'cash',
-    'google pay',
-    'amazon pay',
-    'apple pay'
-  ]
-};
-
-export const deliveryType = {
-  title: 'Delivery Type',
-  name: 'deliveryType',
-  value: ['', 'currier', 'post', 'delivery servise']
-};
 
 export const socialNetworkLinks = {
```

Comment:
It is constant. Please, use the uppercased name

Reply:
deleted - it was unused